README.md

```diff
@@ -16,6 +16,16 @@ git submodule add -b headers-only https://github.com/banditcpp/snowhouse snowhou
 git submodule update --init --recursive
 ```
 
+CMake (>= 3.0) users can use the provided library target:
+```CMake
+shell$ git clone https://github.com/banditcpp/snowhouse.git
```

Comment:
*I think I would prefer the docs like this:*

As an alternative, CMake users can use Snowhouse with the provided library target.
Assuming you have cloned the `master` branch into a `snowhouse` subdirectory,
your `CMakeLists.txt` might contain lines like the following:
```CMake
add_subdirectory(snowhouse)
add_executable(app main.cpp)
target_link_libraries(app snowhouse)
```

Reply:
Sure, I guess this will be easier to understand by people who don't really speak CMake.